scripts/deploy_schema.sh

```
@@ -0,0 +1,209 @@
+echo "    - start_print_session()"
+echo "    - get_active_print_session()"
+echo "    - add_print_message()"
+echo "    - end_print_session()"
+echo ""
+
+# ───────────────────────────────────────────────────────────────────────────
+# 3. Confirm Deployment
+# ───────────────────────────────────────────────────────────────────────────
+
+echo -e "${YELLOW}⚠️  This will deploy the schema to your Neon database.${NC}"
+echo "   Database: $(echo $NEON_DB_URL | sed 's/postgresql:\/\/\([^:]*\):\([^@]*\)@\(.*\)/\3/')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)